### PR TITLE
JolokiaCollector: collect from multiple instances

### PR DESCRIPTION
**Pull request by mfriedenhagen**
This patch allows to collect data from multiple jolokia instances. It follows the syntax in the redisstat collector.
I wrote a couple of tests just to ensure the instances variable is parsed correctly and tested this locally with
a tomcat and just collected the same data twice.

```
enabled = True
instances = mytomcat@localhost:20094, mytomcat2@localhost:20094
```

Sample output in `archive.log`:

```
servers.vagrant-ci-setup-mifr.jmx.mytomcat.java.lang.type_Threading.DaemonThreadCount 31 1424556475
servers.vagrant-ci-setup-mifr.jmx.mytomcat.java.lang.type_Threading.CurrentThreadCpuTime 12729312069 1424556475
servers.vagrant-ci-setup-mifr.jmx.mytomcat.java.lang.type_Threading.TotalStartedThreadCount 58 1424556475
servers.vagrant-ci-setup-mifr.jmx.mytomcat2.java.lang.type_OperatingSystem.OpenFileDescriptorCount 74 1424556483
servers.vagrant-ci-setup-mifr.jmx.mytomcat2.java.lang.type_OperatingSystem.SystemLoadAverage 0 1424556483
servers.vagrant-ci-setup-mifr.jmx.mytomcat2.java.lang.type_OperatingSystem.FreeSwapSpaceSize 0 1424556483
```

When instances is not given `host` and `port` will be taken into account as before.

### CODE REVIEW

File: src/collectors/jolokia/jolokia.py [hunk 1]
```diff
@@ -60,6 +68,9 @@ def get_default_config_help(self):
             "stats. If not provided, all stats will be collected",
             'host': 'Hostname',
             'port': 'Port',
+            'instances': "Jolokia addresses, comma separated, syntax:"
+            + " nick1@host:port, nick2@host:port"
```
Comment (deejay1) [inline src/collectors/jolokia/jolokia.py]: Instead of concatenating the string literal you can do:
```python
'instances': ("Jolokia addresses, comma separated, syntax:"
              " nick1@host:port, nick2@host:port")
```

File: src/collectors/jolokia/test/testjolokia.py [hunk 2]
```diff
@@ -82,8 +102,8 @@ def test_should_allow_interpretation_of_list_values(
             self, interpret_bean_with_list_mock, publish_mock):
         self.collector.collect_bean('prefix', {
             'RecentWriteLatencyMicros': 100,
-            'RecentReadLatencyHistogramMicros': [1, 2, 3]
-        })
+            'RecentReadLatencyHistogramMicros': [1, 2, 3],
+        }, "")
```
Comment (josegonzalez) [inline src/collectors/jolokia/test/testjolokia.py]: Why this change?

Comment (mfriedenhagen) [inline src/collectors/jolokia/test/testjolokia.py]: Because the `instance_prefix` needs to be "" in recursive calls.
